tests/test_entropy.py

```diff
@@ -0,0 +1,26 @@
+"""
+Testing entropy.py functionality
+"""
+
+import pathlib
+
+from test_git_parser import get_most_recent_commits
+
+from almanack.entropy import calculate_shannon_entropy
+
+
+def test_calculate_shannon_entropy(repository_paths: dict[str, pathlib.Path]) -> None:
+    """
+    Test calculate_shannon_entropy function.
+    """
+    file_sets = {
+        "high_entropy": ["high_entropy2.md", "high_entropy.md"],
+        "low_entropy": ["low_entropy.md"],
+    }
+    for label, repo_path in repository_paths.items():
+        source_commit, target_commit = get_most_recent_commits(repo_path)
+        entropies = calculate_shannon_entropy(
+            repo_path, source_commit, target_commit, file_sets[label]
```

Comment:
Is `repo_path` a file path within a repo? If so, consider highlighting this within the name of the variable name and possibly within the parameter name as well. I get the feeling that "repo_path" could be misunderstood to be a path to the root of a repo as opposed to a single file within a repo.

Reply:
I believe that `repo_path` is the path to the root of the repo. Since we loop through the fixture `repository_paths` where the label is the name of the repo and the `repo_path` is the desired path. Please let me know what you think of this!

Reply:
Sounds good!